tests_phi/testcases/data_profiles.py

```diff
@@ -0,0 +1,90 @@
+from tests_phi.testcases.test_base import TestBase
+from tests_phi.pages.login_page import LoginPage
+from tests_phi.pages.dashboard_page import DashboardPage
+from tests_phi.utils import constants
+from tests_phi.utils import browser_helper
+from tests_phi.pages.data_profiles_page import DataProfilesPage
+from tests_phi.pages.general_setting_pages import DAGeneralSettingsPage
+import logging
+import pytest
+
+class Test_DataProfiles(TestBase):
+    
+    login_page = LoginPage()
+    dashboard_page = DashboardPage()
+    data_profiles_page = DataProfilesPage()
+    da_general_setting_page = DAGeneralSettingsPage()
+    
+    data_test = [
+        ("",constants.EMPTY_PROFILE_NAME_MESSAGE),
+        (" /:*?<>|\"#[]{}=%;",constants.PROFILE_NAME_INCLUDE_INVALID_CHAR_MESSAGE),
+        ("Test Case Execution", constants.PROFILE_NAME_EXIST_MESSAGE)]
+    
+    @pytest.mark.usefixtures("clean_up")   
+    @pytest.mark.parametrize("profile_name,err_message", data_test)
+    def test_da_dp_tc069_tc070_tc071(self, profile_name, err_message):
+        ''' - Verify that user is unable to proceed to next step or finish creating data profile if  "Name *" field is left empty
+            - Verify that special characters ' /:*?<>|"#[]{}=%; 'is not allowed for input to "Name *" field
+            - Verify that Data Profile names are not case sensitive
+        '''     
+        logging.info("1. Log in Dashboard")
+        self.login_page.go_to()
```

Comment:
Can you make the test run faster? Its good if we don't need to login/logout on each test data

Reply:
Optimized this suite to reduce running time